exercises/nucleotide-count/tests/Tests.elm

```diff
@@ -10,16 +10,19 @@ tests =
     describe "NucleotideCount"
         [ test "the solution is for the correct version of the test" <|
             \() -> Expect.equal 2 version
```

Comment:
I had noticed the version function on another exercise, but I don't think there was a test for it. What's the story with that?